.github/workflows/test-coverage.yml

```diff
@@ -236,6 +236,27 @@ jobs:
           coverage-artifact-name: "coverage_encryption"
           coverage-path: coverage.txt
 
+  # This job runs the database with telemetry tests using default configuration, on linux.
+  test-telemetry:
+    name: Test telemetry job
+
+    runs-on: ubuntu-latest
+
+    env:
+      GO_FLAGS: -tags=telemetry
```

Comment:
thought: This is a neat way to pass go flags using env variables. Do you think this is more conventional that what we do with other testing types like `DEFRA_CLIENT_GO` for example. IMO if we choose this over `DEFRA_TELEMETRY` might be worth making other flags consistent 

question: What is the `tags` flag?

Reply:
Ah! never mind thanks for clarifying